athenet/network.py

```diff
@@ -2,45 +2,69 @@
 
 import timeit
 import numpy as np
+from enum import Enum
 
 import theano
 import theano.tensor as T
 
 from athenet.layers import WeightedLayer, ConvolutionalLayer
+from athenet.utils import overwrite, save_data_to_pickle
+
+
+class TrainConfig(object):
+    """Class storing information about training process."""
+    def __init__(self):
+        self.n_epochs = None
+        self.batch_size = None
+
+        self.learning_rate = None
+        self.momentum = 0.
+        self.weight_decay = 0.
+
+        self.val_interval = 1
+        self.val_units = 'epochs'
```

Comment:
We don't know if there is any other possibility instead of epochs. More generally, it must be documented, what can the parameters be. Which ones are default, which require to be set? There could be also some correctness check for setting 'val_units' parameter, but not required.